dashboard/test/controllers/datablock_storage_controller_test.rb

```diff
@@ -621,26 +632,26 @@ def create_shared_table
   end
 
   test "import_csv" do
-    CSV_DATA = <<~CSV
+    csv_data = <<~CSV
```

Comment:
Sorry for the noise on these, I guess CAPS means its a class level constant, and ruby was generating warnings, this makes the warnings go away